### ISSUE TITLE
Use of Non-Standard CRS Names

### ISSUE THREAD

**Issue by AlpoTurunen** [luomus/laji#556]
As a user downloading or inputting occurrences into our systems, it might be difficult to understand coordinate reference systems as we are not using standard naming conventions. 

Terms like "euref-fin" are unclear, as they do not specify a projection or coordinate order. There are multiple coordinate systems with the same or similar names. Actually "euref-fin" doesn't even refer to a coordinate system, it's only a datum.

My suggestion is to use standardized CRS names with EPSG codes to avoid possible errors:

- euref-fin -> ETRS-TM35FIN (EPSG:3067)
- wgs84 -> WGS84 (EPSG:4326)
- ykj -> KKJ / Suomen yhtenäiskoordinaatisto / Finland Uniform Coordinate System / (EPSG:2393)

**Comment by mikkohei13** [comment]
For usability, the names need to be understandable in a short form. E.g. I would see them most often on Vihko Excel import files, which could show the first ~5 characters like this:

![image](https://github.com/user-attachments/assets/2d326554-2664-4e0c-a9dd-27ff572a5a0c)

YKJ is standard abbreviation for biologists, so it should be the primary name for this coordinate type.

So if we change these, I would suggest name like "YKJ (EPSG:2393)" or "YKJ yhtenäiskoordinaatisto (EPSG:2393)"

I don't know would this kind of change require any development work? Might be possible by just 

- updating labels on triplestore
- updating test scripts
- updating and improving documentation
- communicating to users

This relates to epics about improving data usability: Obs download fieldnames, Field expansion, API v1.

**Comment by AlpoTurunen** [comment]
Yes, YKJ is good as well. I edited it to my original message. And this is not super important, just nice to have :)
